Gemfile

```diff
@@ -1,7 +1,6 @@
-# encoding: utf-8
-
 source 'https://rubygems.org'
 
 gemspec name: 'mutant'
 
 gem 'devtools', git: 'https://github.com/rom-rb/devtools.git'
+eval_gemfile File.expand_path('../Gemfile.shared', __FILE__)
```

Comment:
There should be a commend in that file explaining the reason why it exist. If its not there anymore I rebased it away by accident.

Always when I update mutants dependencies (especially morpher and anima) there is a state where dependencies need to be pulled from git sources.

These git sources need to be reflected in the Gemfiles for:

* The vendored test app, per rspec (and future minitest) integration
* The corpus tests Gemfiles

That would be ~5 Gemfiles (tendency to grow) I'd need to manually add (remove) the git source from/to.

Now instead: I can simply use this (now empty) file to ease upgrades of shared dependencies.

As I said: there was a note in the `Gemfile.shared` trying to explain its presence as (now empty) file.